tests/test_get_collection_names.py

```
@@ -0,0 +1,205 @@
+        finally:
+            await client.close()
+
+    @pytest.mark.asyncio
+    async def test_property_is_async(self):
+        # The async facade's get_collection_names is a coroutine function.
+        # If a maintainer accidentally makes it sync, this test catches it.
+        client = AsyncQdrantClient(":memory:")
+        try:
+            assert hasattr(client.get_collection_names, "__await__") or hasattr(
+                client.get_collection_names, "__call__"
+            )
```

Ruff (0.16.0)

[warning] Using `hasattr(x, "__call__")` to test if x is callable is unreliable. Use `callable(x)` for consistent results.

Replace with `callable()`

(B004)